check_sites.py

```diff
@@ -25,20 +25,18 @@
 def parse_rws_json(rws_json_string, strict_formatting):
     """Attempts to parse `rws_json_string` as JSON and validate formatting if `strict_formatting` is true.
 
-    Returns a tuple of the JSON dict and None if there were no errors,
-    or None and the error message if there was an error.
+    Returns a tuple of the JSON dict and None if there were no formatting
+    errors, or None and the error message if there was an error.  If the file
+    is not proper json, `json.loads()` will throw a JSONDecodeError.
 
     Args:
         rws_json_string: string
         strict_formatting: bool
     Returns:
         Tuple[Dict|None, string|None]
     """
-    try:
-        rws_sites = json.loads(rws_json_string)
-    except Exception as inst:
-        # If the file cannot be loaded, we will not run any other checks
-        return (None, f"There was an error when parsing the JSON;\nerror was:  {inst}")
+    # If this fails, it needs to be caught in the caller.
```

Comment:
I don't think we need this comment, since we've added an equivalent notice in the docstring.